### DISCUSSION TITLE
Trying to make a button to go to the home page (index.html), but it redirects to the domain I removed already...

### DISCUSSION THREAD

**Discussion by trukum14** [community/community#139732] [Programming Help] [answered]
### Body

Here is the piece of code:
 <div class="container">
        <a href="index.html" class="left">Home</a>
        <a href="AboutUs.html" class="right">About Us</a>
   </div>

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by EugeneBachura** [comment, marked as answer]
### Example of Correct HTML Button (Relative Path)
Here’s a clean example to make sure it redirects locally to index.html:
```html
<button onclick="window.location.href='/index.html';">Go to Home</button>
``` 
Example of Correct HTML Link:
```html
<a href="/index.html">Go to Home</a>
``` 

### Clearing Cache
It's possible that the data, including old links, may be stored in the application's or browser's cache. Clearing the cache can help resolve issues where outdated URLs are being used, as the browser or app may be loading saved versions of the page.


#### If this doesn’t resolve the issue, could you provide more context, such as whether you're working with any JavaScript frameworks, or if you are using any specific server-side technologies? This would help in narrowing down the potential causes further.

**Reply by trukum14** [reply]
Testing it on my iPad, and now it works somehow...

**Reply by EugeneBachura** [reply]
No problem at all. It was most likely just the cache that cleared over time, allowing the new link to start working.
Best of luck with the rest of your project!

**Reply by trukum14** [reply]
Thank you :)
